src/pages/billing/CreateAzureBillingProjectModal.js

```diff
@@ -126,7 +126,7 @@ const CreateAzureBillingProjectModal = ({ onSuccess, onDismiss, billingProjectNa
             h(Link, {
               href: 'https://portal.azure.com/',
               ...Utils.newTabLinkProps
-            }, ['Go to the Azure Portal'])
+            }, ['Go to the Azure Portal.'])
```

Comment:
Adding a period for consistency with the messaging about cost. Even though neither is a complete sentence, I think leaving off the period does look a little odd.

![image](https://user-images.githubusercontent.com/484484/214386165-de9f2f24-9029-44ce-a68a-2604168aa652.png)